src/aosm/azext_aosm/_configuration.py

```diff
@@ -473,6 +328,189 @@ def validate(self):
             )
 
 
+NFD_NAME = (
```

Comment:
These things are all in DESCRIPTION_MAP for other classes. You may have a reason for wanting them out of there for clarity, but tis a bit inconsistent.

Reply:
I pulled them out here for 2 reasons:
- I was finding the description map really unwieldy as it got bigger
- Some of the parameter names were the same as for creating NFDs, but I wanted different descriptions.

I get what you are saying about inconsistency, but on balance I'm going to leave it as it is.